src/Workspaces/CSharpTest/Formatting/FormattingTests.cs

```diff
@@ -9686,7 +9686,7 @@ public async Task SpacingAfterAttribute_Multiple()
         {
             var code = @"class C
 {
-    void M([My][My]  int x)
+    void M([My] [My]  int x)
```

Comment:
please don't change the inputs to existing tests.  please add a new test in a case like this.